src/styles/main.less

```diff
@@ -465,6 +465,7 @@ img {
 
 *::-webkit-scrollbar-track
 {
+	box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
```

Comment:
Not entirely sure it makes a difference, but shouldn't the vendor prefixed version go above this, so the browser uses the unprefixed version if it knows how?

Reply:
Doesn't look like there is any others types:
<https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar>
<https://www.w3schools.com/howto/howto_css_custom_scrollbar.asp>
Seems to be a webkit only thing